Training/Training/Change.c

```diff
@@ -0,0 +1,40 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// Change.c
+// Program on main branch.
+// ------------------------------------------------------------------------------------------------
+#include <stdio.h>
+
+void ChangeRequired (int* balance, int* change10, int* change5, int* change2, int* change1) {
+   int num = *balance;
+   if (num >= 10) {
+      *change10 = num / 10;
+      num %= 10;
+   }
+   if (num >= 5) {
+      *change5 = num / 5;
+      num %= 5;
+   }
+   if (num >= 2) {
+      *change2 = num / 2;
+      num %= 2;
+   }
+   if (num >= 1) {
```

Comment:
What is the purpose of this if statement? What is the remainder of any number divided by 1?